pywinauto/win32defines.py

```diff
@@ -12010,7 +12010,7 @@
 MCMV_CENTURY = 3
 
 MCM_SETCALID = 0 # Variable c_int
-MCM_GETCALID = 1 # Variable c_int
+MCM_GETCALID = 0 # Variable. c_int

```

Comment:
It seems `MCM_GETCALID` must equal to `MCM_FIRST + 25` ([proof link](http://www.rdos.net/svn/tags/V9.2.5/watcom/bld/sdk/spy/spyexmsg.h)). Almost the same for `MCM_SETCALID`.